docs/Navigation/SubscribeWayPoints/index.md

```diff
@@ -0,0 +1,75 @@
+## SubscribeWayPoints
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Subscribe to periodic navigation waypoint/destination updates
+
+### Request
+
+!!! MUST
+Send future waypoint updates to SDL using [Navigation.OnWayPointChange](../onwaypointchange)
+!!!
+
+#### Parameters
+
+This RPC has no additional parameter requirements
```

Comment:
```suggestion
There are no defined parameters for this function
```

Reply:
This is the language used throughout the rest of the docs, so I'm inclined toward keeping it consistent. I also believe that it is clear enough as it is worded now.

Reply:
Additional parameter requirements makes it sound like there are some base parameters which are always required. But if it's the convention then I guess it works